packages/rum-core/src/domain/view/viewMetrics/trackCumulativeLayoutShift.ts

```diff
@@ -76,19 +63,10 @@ export function trackCumulativeLayoutShift(
           if (
             isExperimentalFeatureEnabled(ExperimentalFeature.WEB_VITALS_ATTRIBUTION) &&
             clsTarget &&
-            clsTarget.parentElement
+            // Check if the CLS target have been removed from the DOM between the time we collect the target reference and when we compute the selector
+            clsTarget.isConnected
```

Comment:
The target might have a `parentElement` and not be connected. It can happen if one of the parents has been detached.